waspc/headless-test/start.js

```diff
@@ -1,31 +1,32 @@
-const cp = require('child_process');
-const readline = require('linebyline');
+const cp = require("child_process");
+const readline = require("linebyline");
 
 function spawn(name, cmd, args, done) {
   const spawnOptions = {
     detached: true,
+    stdio: ["ignore", "pipe", "pipe"],
   };
   const proc = cp.spawn(cmd, args, spawnOptions);
 
   // We close stdin stream on the new process because otherwise the start-app
   // process hangs.
   // See https://github.com/wasp-lang/wasp/pull/1218#issuecomment-1599098272.
-  proc.stdin.destroy();
+  // proc.stdin.destroy();
```

Comment:
I need to revert this, this is only needed for headless-test to work on Mac. I haven't yet figured it out.